src/providers/DataProvider.js

```diff
@@ -18,9 +21,10 @@ const DataProvider = ({
 
   const { commonData, isCommonDataLoading } = useCommonData();
   const { classificationBrowseConfig, isLoading: isBrowseConfigLoading } = useClassificationBrowseConfig();
+  const { instanceDateTypes, isLoading: isInstanceDateTypesLoading } = useInstanceDateTypes();
```

Comment:
The date types will be used for the find-instance plugin too, let's move this hook to `useCommonData`.